dotCMS/src/main/java/com/dotmarketing/portlets/structure/business/FieldAPI.java

```diff
@@ -54,6 +55,13 @@
 	 * @return
 	 */
 	public boolean isElementConstant(Field field);
+
+	/**
+	 * A field that is a hidden
+	 * @param field
+	 * @return
+	 */
+	public boolean isElementHidden(Field field);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Field"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)